client/css/style.css

```diff
@@ -678,9 +678,15 @@ button, .user {
 }
 #chat .part .from:before,
 #chat .quit .from:before {
-    font-family: FontAwesome;
+	font-family: FontAwesome;
 	content: "\f08b";
 	color: #FF4136;
+	display: inline-block;
+	-webkit-transform: rotate(180deg);
+	-moz-transform: rotate(180deg);
+	-ms-transform: rotate(180deg);
+	-o-transform: rotate(180deg);
+	transform: rotate(180deg);
```

Comment:
I was thinking about that this morning actually, glad someone stepped up :)
Completely safe to remove the prefix-vendor versions here.
The spec has been stagnant for more than 2 years (in WD state, which is a bit worrying actually), so browsers have definitely had time to catch up with this. Plus, s***w old browsers :-)